src/neuroconv/datainterfaces/behavior/lightningpose/lightningposeconverter.py

```diff
@@ -133,6 +144,20 @@ def add_to_nwbfile(
         stub_test : bool, optional
             If True, only a subset of the data will be added for testing purposes, by default False.
         """
+        # Deprecate external_mode parameter
+        # TODO: Remove after May 2026
+        if external_mode is not True:
+            warnings.warn(
+                "The 'external_mode' parameter is deprecated and will be removed in May 2026. "
+                "It will be replaced with a choice between ExternalVideoInterface and InternalVideoInterface.",
```

Comment:
On or after May 2026, only external videos will be supported by the LightningPoseConverter.